CMakeLists.txt

```diff
@@ -29,7 +29,11 @@ find_package(Clang CONFIG)
 set(JAKT_CPP_AUTO_IMPORT_PROCESSOR_DEFAULT none)
 if (Clang_FOUND)
   set(JAKT_CPP_AUTO_IMPORT_PROCESSOR_DEFAULT clang)
-  find_program(CLANG_PATH clang REQUIRED)
```

Comment:
Have a look at the ClangConfig cmake file in your clang install. are there any variables there that point to the clang binary dir, or clang executable?

Reply:
Yeah, thanks!

I had actually been doing that when trying to make this MR. :thonk:

I can't immediately see anything super nice to use for this. There is `CLANG_INSTALL_PREFIX` which points to: `/usr/lib/llvm-17`, but then I need to know where clang is installed past that. The best option I was thinking of to try out would be to use a clang binary target from `/usr/lib/llvm-17/lib/cmake/clang/ClangTargets.cmake`, but I hadn't / haven't quite worked out how to do that yet.